src/SourceBuild/content/Directory.Build.targets

```diff
@@ -2,14 +2,18 @@
 
   <Import Project="Sdk.targets" Sdk="Microsoft.DotNet.Arcade.Sdk" Condition="'$(SkipArcadeSdkImport)' != 'true'" />
 
-  <Target Name="DetermineSourceBuiltSdkVersion">
+  <Target Name="GetDotNetSdkTarball">
```

Comment:
Nit: `GetDotNetSdkArchive` to be archive extension agnostic.

Reply:
Fixed in https://github.com/dotnet/installer/pull/19222/commits/b74d5fb4c0680ad2022d724d0e00e68585689c19